src/gei/Commands/MigrateArchiveRepoCommand.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter.Commands
+{
+    public class MigrateArchiveRepoCommand : Command
+    {
+        private readonly OctoLogger _log;
+        private readonly ISourceGithubApiFactory _sourceGithubApiFactory;
+        private readonly ITargetGithubApiFactory _targetGithubApiFactory;
+        private readonly IAzureApiFactory _azureApiFactory;
+        private readonly EnvironmentVariableProvider _environmentVariableProvider;
+
+        private const int Archive_Generataion_Timeout_In_Hours = 10;
+        private const int Check_Status_Delay_In_Milliseconds = 10000; // 10 seconds
+        private const string Git_Archive_File_Name = "gitArchive.tar.gz";
+        private const string Metadata_Archive_File_Name = "metadataArchive.tar.gz";
```

Comment:
`const`s should be all caps and snaked case --> `THIS_IS_A_CONSTANT`

Reply:
Updated in https://github.com/github/gh-gei/pull/221/commits/3b1c55ff051c5481c58c6db30f932e4e8d58b675